lib/soft_timer_delay.h

```diff
@@ -77,11 +77,8 @@
 
 #include <stdint.h>
 
-enum
-{
-    /** Standard timer frequency, corresponding to a 32kHz clock */
-    SOFT_TIMER_FREQUENCY = 32768,
-};
+/** Standard timer frequency, corresponding to a 32kHz clock */
+#define SOFT_TIMER_FREQUENCY 32768
```

Comment:
Are there any possible side-effects somewhere else ?
I don't think but I may be missing some corner case.

Also in contiki code you used it as

`#define RTIMER_ARCH_SECOND (SOFT_TIMER_FREQUENCY)`

Is there any reason for the parenthesis for constants ?
Should they be added here by default also ?

Is it just because it's good practice for macros in general or is there a reason for doing it around a constant ?